fabulaws/library/wsgiautoscale/templates/nginx.conf

```diff
@@ -32,7 +32,9 @@ server {
         location = /healthcheck.html {
             # Turn off basic auth for healthcheck
             auth_basic "off";
-            proxy_pass http://django_server;
+            # Abbreviated version of logic below to serve healthcheck.html (override) if it exists
+            if (-f $request_filename) { break; }
+            if (!-f $request_filename) { proxy_pass http://django_server; break; }
```

Comment:
This seems like a verbose way of writing, but I suspect that it's achieving some end with nginx. But I'm still curious: why wouldn't this work?

```
location = /healthcheck.html {
    auth_basic "off";
    if (!-f $request_filename) { proxy_pass http://django_server; }
    break;
}
```